swift/ql/lib/upgrades/147e087e57e51b2eb41e75c9c97380d0e6c20ecb/swift.dbscheme.stats

```diff
@@ -0,0 +1,4 @@
+<dbstats>
+     <typesizes />
+     <stats />
+</dbstats>
```

Comment:
We didn't put stats in any other upgrade scripts, do we need this now?

Reply:
I don't think we do, no.